Dockerfile

```diff
@@ -123,7 +128,9 @@ ENV THRIFT_DEPS automake \
                 libtool \
                 pkg-config \
                 python \
-                python-dev
+                python-dev \
+                python-pip \
```

Comment:
How come that none of the previous packages have a dependency on python-pip, yet they are using it, and here you do have it as a depenency?

Reply:
They already had it as a dependency so they could install the python packages they depended on, but this was a case where we hadn't needed it until now.